phys/module_radiation_driver.F

```diff
@@ -1647,6 +1648,26 @@ SUBROUTINE radiation_driver (                                  &
 
              CALL wrf_debug(100, 'CALL NUWRF goddard longwave radiation scheme 2017')
 
+            IF ( mp_physics .NE. nuwrf4icescheme ) THEN
+                IF ( has_reqc .EQ. 1 ) THEN
+                   DO j=jts,jte
+                   DO k=kts,kte
+                   DO i=its,ite
+                      re_cloud_gsfc(i,k,j) = re_cloud(i,k,j) * 1.E+6
+                      re_ice_gsfc(i,k,j) = re_ice(i,k,j) * 1.E+6
+                      re_snow_gsfc(i,k,j) = re_snow(i,k,j) * 1.E+6
+                      re_rain_gsfc(i,k,j) = 0.
+                      re_graupel_gsfc(i,k,j) = 0.
+                      re_hail_gsfc(i,k,j) = 0.
+                   ENDDO
+                   ENDDO
+                   ENDDO
+                ELSE
+                   WRITE ( wrf_err_message , * ) 'must choose one of the microphysics: 3,4,6,8,14,16,17-21,28,51,or 52'
```

Comment:
maybe it could say must choose a microphysics option with an effective radius - not sure if the list is necessary, but it seems the User Guide should put this info into the microphysics table.

Reply:
@dudhia We can do that.